cmd/tailscale/cli/file.go

```diff
@@ -286,22 +287,106 @@ func runCpTargets(ctx context.Context, args []string) error {
 	return nil
 }
 
+type onConflict string
+
+const (
+	skipOnExist         onConflict = "skip"
+	overwriteExisting              = "overwrite" //  Overwrite any existing file at the target location
```

Comment:
This doesn't do what you think it does: https://go.dev/play/p/587SMC5Rn-K

Reply:
rightie oh. thanks.